drivers/hmc7044/hmc7044.c

```diff
@@ -235,6 +236,33 @@ static int hmc7044_write(struct hmc7044_dev *dev,
 	return spi_write_and_read(dev->spi_desc, buf, ARRAY_SIZE(buf));
 }
 
+/**
+ * SPI register read from device.
+ * @param dev - The device structure.
+ * @param reg - The register address.
+ * @param val - The register data.
+ * @return SUCCESS in case of success, negative error code otherwise.
+ */
+static int hmc7044_read(struct hmc7044_dev *dev,
```

Comment:
From this single commit PR i don't see the need for this extension. Usually in a PR you should also add the use of these functions that you created or at least say in the commit message that they are intended for future use.